Source/WebCore/layout/formattingContexts/inline/text/TextUtil.cpp

```diff
@@ -220,10 +220,10 @@ TextUtil::EnclosingAscentDescent TextUtil::enclosingGlyphBoundsForText(StringVie
         return { };
 
     if (textContent.is8Bit()) {
-        auto textIterator = Latin1TextIterator { textContent.characters8(), 0, textContent.length(), textContent.length() };
+        auto textIterator = Latin1TextIterator { textContent.span8().data(), 0, textContent.length(), textContent.length() };
         return enclosingGlyphBoundsForRunWithIterator(style.fontCascade(), !style.isLeftToRightDirection(), textIterator);
     }
-    auto textIterator = SurrogatePairAwareTextIterator { textContent.characters16(), 0, textContent.length(), textContent.length() };
+    auto textIterator = SurrogatePairAwareTextIterator { textContent.span16().data(), 0, textContent.length(), textContent.length() };
```

Comment:
Seems like these iterators should be taking spans; I know we say this all the time when reviewing these.